index.js

```diff
@@ -13,6 +13,7 @@ const authors = {
   hueniverse: 'https://www.patreon.com/eranhammer',
   hughsk: 'https://hughsk.io/donate/',
   jayphelps: 'https://www.patreon.com/jayphelps',
+  johnleider: 'https://www.patreon.com/vuetify',
```

Comment:
This npm user has no packages: https://www.npmjs.com/~johnleider

It looks like you typo'd your username: https://www.npmjs.com/~johnjleider